src/shared/util/verification.ts

```diff
@@ -21,3 +21,11 @@ export enum VfnErrors {
   TransactionNotFound = 'TRANSACTION_NOT_FOUND',
   InvalidMobileNumber = 'INVALID_MOBILE_NUMBER',
 }
+
+export enum ADMIN_VERIFIED_SMS_STATES {
+  limitExceeded = 'LIMIT_EXCEEDED',
+  belowLimit = 'BELOW_LIMIT',
+  hasMessageServiceId = 'MESSAGE_SERVICE_ID_OBTAINED',
+}
+
+export const SMS_VERIFICATION_LIMIT = 10000
```

Comment:
can this be an env var (that is then sent to the frontend the usual env-var-ry way)

Reply:
sure, could i ask for clarification on when we decide something should be an env-var vs a constant?